src/main/java/bio/terra/cli/command/config/set/AppLaunch.java

```diff
@@ -18,6 +20,15 @@
   protected void execute() {
     Config config = Context.getConfig();
     Config.CommandRunnerOption prevAppLaunchOption = config.getCommandRunnerOption();
+    if (CommandRunnerOption.DOCKER_CONTAINER == mode) {
+      String imageId = Context.getConfig().getDockerImageId();
+      if (!new DockerClientWrapper().checkImageExists(Config.getDefaultImageId())) {
+        OUT.printf(
+            "WARNING: Either the Docker daemon isn't running or image %s was not found on the local machine. "
+                + "Do `docker pull %s` to obtain it.%n",
```

Comment:
If Docker isn't running or installed, then I think `checkImageExists` will throw an exception, not return false. You can check that by turning off Docker on your machine and trying to change the image id.